libr/asm/asm.c

```diff
@@ -1098,15 +1098,15 @@ R_API RAsmCode* r_asm_rasm_assemble(RAsm *a, const char *buf, bool use_spp) {
 	if (use_spp) {
 		Output out;
 		out.fout = NULL;
-		out.cout = r_strbuf_new ("");
-		r_strbuf_init (out.cout);
+		out.cout = (SStrBuf *)r_strbuf_new ("");
+		r_strbuf_init ((RStrBuf *)out.cout);
```

Comment:
ideally, spp should have a Bind-like struct to replace this apis at runtime, or maybe compile it like we did before with an ifdef to avoid using that s_ api and use just r_ with some macros